### ISSUE TITLE
Detection accuracy per image 

### ISSUE THREAD

**Issue by florence27** [facebookresearch/maskrcnn-benchmark#1101]
## ❓ Questions and Help
I was wondering, if there's any way to get the detection accuracy per image, while testing the network? Would be nice to see in which cases the network fails.

**Comment by Darshan2701** [comment]
@florence27 Without using the ground truth as a reference you cannot get information on the accuracy of the model. Once the training is done, the model is tested on the validation images and if you have only one validation image with its corresponding ground truth, you can get the detection accuracy simply after training or using "test_net.py". 
Ideally, once you are satisfied with the validation accuracy, it is a good idea to use the "run_on_opencv_images" function in the predictor.py to get the predictions and check it manually.

**Comment by florence27** [comment]
@Darshan2701 thanks for your answer. Ok, but suppose I have the ground truth (which we do have, from the labels), how can I evaluate the model on one single image? I mean which function would I have to apply, because I tried several different things, like changing the dataLoader object to load only one single image, but I didn't manage to get evaluation results for this single image. Do you have any idea, how I could accomplish that?

**Comment by Darshan2701** [comment]
@florence27 well the COCO eval script does the evaluation based on the "predictions" obtained from the "inference.py"'s function "compute_on_dataset". I believe that just by having a single image with its ground-truth in the validation set would give you the results by executing "test_net.py". But this wouldn't be an elegant solution.
